precompiles/erc20/types_test.go

```diff
@@ -0,0 +1,230 @@
+package erc20_test
+
+import (
+	"math/big"
+
+	"github.com/evmos/evmos/v15/precompiles/erc20"
+	utiltx "github.com/evmos/evmos/v15/testutil/tx"
+)
+
+func (s *PrecompileTestSuite) TestParseTransferArgs() {
```

Comment:
Do we really need the entire test suite here? 
https://github.com/evmos/evmos/blob/main/precompiles/outposts/osmosis/types_test.go#L78

Reply:
What's the benefit of not using it? The resulting tests would have a different syntax compared to the other tests in the package, so I don't see any good in that

Reply:
The benefit is that it will be a real unit test, not a unit test that is using an integration suite just to test a function call that doesn't need the entire setup of a chain.

Reply:
The tests are not setting up the chain though?

Reply:
The tests are not setting up the chain though?